go/proxyd/backend.go

```diff
@@ -316,7 +325,16 @@ func (b *Backend) doForward(ctx context.Context, rpcReq *RPCReq) (*RPCRes, error
 		httpReq.SetBasicAuth(b.authUsername, b.authPassword)
 	}
 
+	xForwardedFor := GetXForwardedFor(ctx)
+	if b.stripTrailingXFF {
+		ipList := strings.Split(xForwardedFor, ",")
```

Comment:
Should be `", "`, since the canonical format is `X-Forwarded-For: <client>, <proxy1>, <proxy2>
`.